Evernote.DataContext/Abstract/DbRepository.cs

```diff
@@ -15,36 +15,33 @@ public abstract class DbRepository<T> : IDbRepository<T> where T : class, IDbEnt
 
         public async Task<T> AddItemAsync(T entity) {
             _dbSet.Add(entity);
-            await SaveChangesAsync();
-
             return entity;
 
         }
 
         public async Task<IEnumerable<T>> AddItemsAsync(IEnumerable<T> entities) {
             await _dbSet.AddRangeAsync(entities);
-            await SaveChangesAsync();
             return entities;
         }
 
         public async Task<bool> ChangeItemAsync(T entity) {
 
 
             _dbSet.Attach(entity).State = EntityState.Modified;
-            return await SaveChangesAsync() > 0;
+            return true;
         }
 
         public async Task<bool> DeleteItemAsync(Guid id) {
             var candidate = await _dbSet.FirstOrDefaultAsync(e => e.Id == id);
             _dbSet.Remove(candidate);
-            return await SaveChangesAsync() > 0;
+            return true;
         }
         public async Task<bool> DeleteItemsAsync(IEnumerable<T> entities) {
             foreach (var item in entities) {
                 var h = _dbSet.Find(item.Id);
                 _dbSet.Remove(h);
             }
-            return await SaveChangesAsync() > 0;
+            return true;
```

Comment:
I don't see the point of returning true if false is never returned. Then it is better to replace the return value of Task<bool> with Task. Then the result of the task will either be successful or the code will throw an exception. This is a cleaner and better approach.